dotcom-rendering/src/lib/decideContainerOverrides.ts

```diff
@@ -43,44 +46,49 @@ const textCardHeadline = (
 const textCardStandfirst = textCardHeadline;
 const textCardFooter = textCardHeadline;
 
-const textContainerFooter = (
-	containerPalette: Extract<DCRContainerPalette, 'Branded'>,
-): string => {
+const textContainerFooter = (containerPalette: DCRContainerPalette): string => {
 	switch (containerPalette) {
-		case 'Branded':
+		case 'BrandedPalette':
+			return neutral[46];
+		default:
```

Comment:
We would usually keep a fall-through behaviour, here… but currently it's not even clear why this isn't simply a constant.